ratis-docs/content/cli.md

```diff
@@ -23,6 +23,37 @@ Ratis-shell is the command line interface of Ratis.
 > Ratis-shell is currently only **experimental**.
 > The compatibility story is not considered for the time being.
 
+
+## Get the ratis-shell source 
+
+>Mention: Ratis shell is available starting from Apache Ratis 2.3.0.
+
+### Build `ratis-shell` from src tarball
+Get the ratis src tarball from [Ratis Source Download](https://ratis.apache.org/downloads.html)
+
+```
+$ tar -C <DST_DIR> -zxvf apache-ratis-*-src.tar.gz
+
+$ cd <DST_DIR>/apache-ratis-*-src
+
+$ mvn -DskipTests -Prelease -Papache-release clean package assembly:single
+...
+[INFO] BUILD SUCCESS
+
+$ tar -C <DST_DIR>/ratis-shell -xzf ratis-assembly/target/apache-ratis-2.3.0-SNAPSHOT-src.tar.gz
+
+$ cd <DST_DIR>/ratis-shell/apache-ratis-2.3.0-SNAPSHOT
+```
+TODO(codings-dan): Get `ratis-shell` from bin tarball directly
```

Comment:
Let's remove this TODO.  Otherwise, it will show up in the doc.

Reply:
I have remove this TODO. I will add this part in the future. So what can I do to make user get `ratis-shell` from bin tarball directly?